roles/matrix-server/templates/riot-web/config.json.j2

```diff
@@ -10,10 +10,24 @@
     "integrations_jitsi_widget_url": "{{ matrix_riot_web_integrations_jitsi_widget_url }}",
     "bug_report_endpoint_url": "https://riot.im/bugreports/submit",
     "enableLabs": true,
+{% if riot_roomdir_servers is defined %}
```

Comment:
If we provide nice defaults in `roles/matrix-server/defaults/main.yml` as discussed, we wouldn't need to have an `is defined` check at all, which will make the code nicer and cleaner